cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -262,7 +262,7 @@
 
     <property>
       <name>stream.bind.address</name>
-      <value>127.0.0.1</value>
```

Comment:
:+1: for defaults being 0.0.0.0.  This will reduce the numerous error cases when a service cant be accessed in a distributed cluster because its listening on localhost only, and its consistent with hadoop (http://hadoop.apache.org/docs/r2.6.0/hadoop-project-dist/hadoop-hdfs/hdfs-default.xml).  Primary downside I can think of would be security related, but as jwang47 mentioned, production environments should be configured anyway.